include/mbedtls/ccm.h

```diff
@@ -288,6 +293,141 @@ int mbedtls_ccm_star_auth_decrypt( mbedtls_ccm_context *ctx, size_t length,
                       const unsigned char *input, unsigned char *output,
                       const unsigned char *tag, size_t tag_len );
 
+/**
+ * \brief           This function starts a CCM encryption or decryption
+ *                  operation.
+ *
```

Comment:
Sorry, I forgot the obvious in my previous review: we need to state explicitly (I think as a doxygen warning) that this function is not implemented in Mbed TLS yet. It will be implemented in a future version, and may be implemented by some `ALT` implementations now, but it's not implemented in Mbed TLS itself right now.

(Same for the other functions declared but not implemented in this PR.)

Reply:
roger that, will do.